examples/js/archive.js

```diff
@@ -1,4 +1,6 @@
 let map;
+let image;
+let imageTooltipText;
```

Comment:
This can be removed if we follow some of the steps I outlined lower in this file to simplify things...

Reply:
Sure... I will, thanks!